<?xml version="1.0" encoding="utf-8"?>
<!-- Launch file for online Relative Deviation analysis -->

<launch>

  <arg name="map_frame" default="/map" />
  <arg name="discrete_color" default="false" />
  <arg name="score_threshold" default="0.0015" />
  <arg name="path" default="$(find relative_deviations)" />
  <arg name="cad_topic" default="mesh_publisher/mesh_out" />
  <arg name="scan_topic" default="ref_corrected_scan" />
  <arg name="map_topic" default="map" />

  <node name="relative_dev"
        pkg="relative_deviations"
        type="relative_deviations_node"
        output="screen"
        respawn="false">
    <rosparam command="load" file="$(find relative_deviations)/launch/parameters.yaml" />
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="discrete_color" value="$(arg discrete_color)" />
    <param name="score_threshold" value="$(arg score_threshold)" />
    <param name="path" value="$(arg path)" />
    <param name="cad_topic" value="$(arg cad_topic)" />
    <param name="scan_topic" value="$(arg scan_topic)" />
    <param name="map_topic" value="$(arg map_topic)" />
  </node>

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find relative_deviations)/launch/rviz.rviz" />-->
</launch>
